README.md

```diff
@@ -1 +1,3 @@
 # This is a simple readme file for mycompany-exercise0 project
+
+# There is a hello.sh script that you can use and run
```

Comment:
looks good but add details on how to execute the script

Reply:
added the necessary information